src/components/CODXCheckoutControl.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div class="checkout-container">
+    <div class="column">
+      <slot></slot>
+    </div>
+    <div class="column">
+      <h3>Summary</h3>
+      <div class="row">
+        <p>{{ action }}</p>
+        <p>{{ cost | formatCODXAmount }}</p>
+      </div>
+      <div class="row">
+        <p>Total</p>
+        <p>{{ cost | formatCODXAmount }}</p>
+      </div>
+      <div class="row">
+        <p>Remaining credits after</p>
+        <p>{{ newBalance | formatCODXAmount }}</p>
+      </div>
+
+      <div v-if="insufficientCODX">
+        <b-button to="/get-codx">
+          Buy more CODX
+        </b-button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'CODXCheckoutControl',
+
+  props: {
+    action: {
+      type: String,
+      required: true,
+    },
+
+    cost: {
+      type: String,
+      required: true,
+    },
+
+    newBalance: {
+      type: Object,
+      required: true,
+    },
+
+    insufficientCODX: {
+      type: Boolean,
+      required: true,
+    },
+  },
+}
+</script>
+
+<style lang="stylus" scoped>
+.checkout-container
+  display: flex
+  flex-direction: row
+  font-size: 80%
+  text-align: left
+
+  .column
+    width: 50%
+
+    .row
+      justify-content: space-between
+      margin: 0
+      width: 80%
+
+      @media screen and (min-width: $breakpoint-sm)
+        flex-direction: column
+        background-color: red
```

Comment:
This seems like a debug line to me

Reply:
Yeah, I was having issues with this breakpoint for some reason. Going to remove it completely and we can tackle it as apart of the style polish.